CMakeLists.txt

```diff
@@ -745,6 +752,16 @@ elseif(IOS)
 
 
 	set(TargetBin PPSSPP)
+elseif(WIIU)
+	set(nativeExtra ${nativeExtra}
+		WiiU/WiiUHost.h
+		WiiU/WiiUHost.cpp
+      WiiU/WiiUMain.cpp
+      WiiU/GX2GraphicsContext.h
+      WiiU/GX2GraphicsContext.cpp
```

Comment:
nit: Spaces and tabs living together in despicable harmony.  Though this file is already a battle ground, I think...

-[Unknown]

Reply:
yeah sorry, easy to miss spaces here since they are all over the place xD